src/internal/utils.js

```diff
@@ -20,19 +20,20 @@ export function check(value, predicate, error) {
 }
 
 export const is = {
-  undef     : v => v === null || v === undefined,
-  notUndef  : v => v !== null && v !== undefined,
-  func      : f => typeof f === 'function',
-  number    : n => typeof n === 'number',
-  array     : Array.isArray,
-  promise   : p => p && is.func(p.then),
-  iterator  : it => it && is.func(it.next) && is.func(it.throw),
-  task      : t => t && t[TASK],
-  observable: ob => ob && is.func(ob.subscribe),
-  buffer    : buf => buf && is.func(buf.isEmpty) && is.func(buf.take) && is.func(buf.put),
-  pattern   : pat => pat && ((typeof pat === 'string') || (typeof pat === 'symbol') || is.func(pat) || is.array(pat)),
-  channel   : ch => ch && is.func(ch.take) && is.func(ch.close),
-  helper    : it => it && it[HELPER]
+  undef       : v => v === null || v === undefined,
+  notUndef    : v => v !== null && v !== undefined,
+  func        : f => typeof f === 'function',
+  number      : n => typeof n === 'number',
+  array       : Array.isArray,
+  promise     : p => p && is.func(p.then),
+  iterator    : it => it && is.func(it.next) && is.func(it.throw),
+  task        : t => t && t[TASK],
+  observable  : ob => ob && is.func(ob.subscribe),
+  buffer      : buf => buf && is.func(buf.isEmpty) && is.func(buf.take) && is.func(buf.put),
+  pattern     : pat => pat && ((typeof pat === 'string') || (typeof pat === 'symbol') || is.func(pat) || is.array(pat)),
+  channel     : ch => ch && is.func(ch.take) && is.func(ch.close),
+  helper      : it => it && it[HELPER],
+  stringableFn: f => is.func(f) && f.hasOwnProperty('toString')
```

Comment:
yeah, basically yes, I know that it is uncommon for people doing that etc and thats just being me too cautious, especially that people could always shoot themselves in the foot (i.e. `Object.prototype.hasOwnProperty` overriding), but when `lodash` do smth I feel confident there is a reason to that and maybe we should apply the same thing here:
[1](https://github.com/lodash/lodash/blob/ef618992b5c206fd07202d6234807085fe3b58d7/lodash.js#1453) [2](https://github.com/lodash/lodash/blob/ef618992b5c206fd07202d6234807085fe3b58d7/lodash.js#L3105)